src/ffi/structseq.rs

```diff
@@ -1,5 +1,6 @@
 use crate::ffi::object::{PyObject, PyTypeObject};
 use crate::ffi::pyport::Py_ssize_t;
+use crate::ffi::tupleobject::{PyTupleObject, PyTuple_GET_ITEM, PyTuple_SET_ITEM};
```

Comment:
```suggestion
#[cfg(not(Py_LIMITED_API))]
use crate::ffi::tupleobject::{PyTupleObject, PyTuple_GET_ITEM, PyTuple_SET_ITEM};
```

Reply:
Hmm, this import is still failing on PyPy. It's probably tidier to remove this and use the definitions inline below?